packages/commons/src/context/context.ts

```diff
@@ -1,66 +1,13 @@
 /* eslint-disable functional/immutable-data */
-import { AsyncLocalStorage } from "async_hooks";
-import { NextFunction, Request, Response } from "express";
 import { zodiosContext } from "@zodios/express";
 import { z } from "zod";
-import { P, match } from "ts-pattern";
-import { AuthData, defaultAuthData } from "../auth/authData.js";
-import { readHeaders } from "../auth/headers.js";
-
-export type AppContext = {
-  authData: AuthData;
-  messageData?: {
-    eventType: string;
-    eventVersion: number;
-    streamId: string;
-  };
-  correlationId?: string | null | undefined;
-};
-export type ZodiosContext = NonNullable<typeof zodiosCtx>;
-export type ExpressContext = NonNullable<typeof zodiosCtx.context>;
```

Comment:
Just moved these two lines below the definition of ctx, and the rest in a new file dedicated to logger context